roles/openshift_common/templates/node-machineset.yml

```diff
@@ -0,0 +1,59 @@
+apiVersion: machine.openshift.io/v1beta1
```

Comment:
This file contains trailing white spaces (spaces at the end of the lines). Can those be cleaned up?
I use this git pre-commit hook which cleans the trailing white spaces for me automatically: https://github.com/noseka1/rcfiles/blob/master/.git.template/hooks/pre-commit

Reply:
my bad, I just enabled whitespace trimming at file save time in VSC.